java/impl/aws/pom.xml

```diff
@@ -87,12 +87,6 @@
             <artifactId>commons-lang3</artifactId>
             <version>${dependency.apache-commons-lang3.version}</version>
         </dependency>
-        <!-- explicit direct dep on apache HTTP Client (psoxy-core has transitive dep on it) -->
```

Comment:
I did remove this, is not used by us anywhere (is a transitive dep of `google-http-client`) but same way we don't specify its other transitive dependencies.
![image](https://user-images.githubusercontent.com/6508426/152606137-c87d2126-80a0-41ff-9c6c-8f7f2c4359dc.png)

If we explicitly use it, we add it. That makes sense.

Reply:
ok, i probably put it here thinking we did, or at some point we did. thanks for the cleanup!